.github/workflows/release.yml

```
@@ -62,47 +62,85 @@ jobs:
           path: rel-notes.md
 
   build_release:
-    needs: release_notes
     name: Build Release
+    needs: release_notes
     strategy:
       matrix:
         platform:
           - name: virtual
-            image: default
-            nodes: [self-hosted, 1ES.Pool=gha-virtual-ccf-sub]
+            os: ubuntu
+            image: ghcr.io/microsoft/ccf/ci/default:build-14-01-2025
+            package_version: DEB
```

```suggestion
            package_format: DEB
```

version sounds a lot like the version number